libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -41,7 +43,7 @@ export class SkyLabelComponent implements OnDestroy, OnInit {
    */
   @Input()
   public set descriptionType(value: SkyIndicatorDescriptionType | undefined) {
-    this.#_descriptionType = value;
+    this.#descriptionType = value;
```

Comment:
What's the reasoning behind removing the `_`? Should we add a getter instead of changing the pattern?

Reply:
I removed it because we reference it outside the setter. This is consistent with other places in the code where we do something similar, but I don't mind adding a getter instead.

Reply:
Fixed.

Reply:
Thanks. I remember a few discussions in the past where we didn't want to use a setter's property anywhere else in the code. I could be misremembering.